consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/validation/ProposalPayloadValidatorTest.java

```diff
@@ -167,30 +167,6 @@ public void validationFailsWhenMessageMismatchesExpectedHeight() {
     verifyNoMoreInteractions(blockValidator);
   }
 
-  @Test
```

Comment:
What about having a test that shows that we now validate Proposal messages independently of the round number in the block?

Reply:
done